### PR TITLE
Revert "Add GraphQL Example to Ballerina-In-Action Section"

### PR DESCRIPTION
**Pull request by praneesha**
Reverts ballerina-platform/ballerina-dev-website#3708

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -404,73 +403,6 @@ <h2>Ballerina in Action</h2>
...
-    private final http:Client httpClient;
-    private final Currency baseCurrency;
-
-    function init() returns error? {
-        self.dbClient = check new (host, user, password, database, port);
-        self.httpClient = check new (apiEndpoint);
-        self.baseCurrency = USD;
-    }
-
-    resource function get search(string id, Currency currency) returns Album|error {
```
Comment (sameerajayasoma) [inline index.html:459]: Is search a proper resource name in GraphQL?

File: index.html [hunk 2]
```diff
@@ -404,73 +403,6 @@ <h2>Ballerina in Action</h2>
...
-        Album album = check self.dbClient->queryRow(`SELECT * FROM Albums WHERE id=${id}`);
-
-        if currency != self.baseCurrency {
-            string query = string `from=${self.baseCurrency}&to=${currency}`;
-            json response = check self.httpClient->get("/currencyExchange?" + query);
```
Comment (sameerajayasoma) [inline index.html:464]: I would use an application-specific record type here.

Comment (sameerajayasoma) [reply index.html:464]: With that we can remove if/else below

File: index.html [hunk 3]
```diff
@@ -404,73 +403,6 @@ <h2>Ballerina in Action</h2>
...
-}
-
-service / on new graphql:Listener(9000) {
-    private final mysql:Client dbClient;
-    private final http:Client httpClient;
-    private final Currency baseCurrency;
-
-    function init() returns error? {
-        self.dbClient = check new (host, user, password, database, port);
-        self.httpClient = check new (apiEndpoint);
```
Comment (sameerajayasoma) [inline index.html:455]: I would use a meaningful variable name instead of `httpClient`.

**Review by sameerajayasoma — CHANGES_REQUESTED** [review]

**Comment by ThisaruGuruge** [comment]
Closing as the comments have been addressed in #3748

**Pull request closed without merging**